kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx

```diff
@@ -22,25 +22,41 @@ describe('Brokers Component', () => {
     );
 
   describe('Brokers', () => {
+    let fetchBrokersMock: fetchMock.FetchMockStatic;
+    const fetchStatsUrl = `/api/clusters/${clusterName}/stats`;
+
+    beforeEach(() => {
+      fetchBrokersMock = fetchMock.getOnce(
+        `/api/clusters/${clusterName}/brokers`,
+        clusterStatsPayload
+      );
+    });
+
     it('renders', async () => {
-      const mock = fetchMock.getOnce(
-        `/api/clusters/${clusterName}/stats`,
+      const fetchStatsMock = fetchMock.getOnce(
+        fetchStatsUrl,
         clusterStatsPayload
       );
       renderComponent();
-      await waitFor(() => expect(mock.called()).toBeTruthy());
+      await waitFor(() => {
+        expect(fetchStatsMock.called()).toBeTruthy();
+        expect(fetchBrokersMock.called()).toBeTruthy();
+      });
       expect(screen.getByRole('table')).toBeInTheDocument();
       const rows = screen.getAllByRole('row');
       expect(rows.length).toEqual(3);
     });
 
     it('shows warning when offlinePartitionCount > 0', async () => {
-      const mock = fetchMock.getOnce(`/api/clusters/${clusterName}/stats`, {
+      const fetchStatsMock = fetchMock.getOnce(fetchStatsUrl, {
         ...clusterStatsPayload,
         offlinePartitionCount: 1345,
       });
       renderComponent();
-      await waitFor(() => expect(mock.called()).toBeTruthy());
+      await waitFor(() => {
+        expect(fetchStatsMock.called()).toBeTruthy();
+        expect(fetchBrokersMock.called()).toBeTruthy();
+      });
```

Comment:
please use sepparate `await waitFor ` for the each expect case

Reply:
done👍